src/main/scala/mesosphere/marathon/core/instance/Goal.scala

```diff
@@ -0,0 +1,54 @@
+package mesosphere.marathon
+package core.instance
+
+import play.api.libs.json.{Format, JsError, JsResult, JsString, JsSuccess, JsValue, JsonValidationError, Reads, Writes}
+
+/**
+  * Defines goal of the instance.
+  * Goal is set by an orchestration layer and interpreted by scheduler layer.
+  * In the end it is used by low-level scheduler to make scheduling decisions e.g. should the task associated with this instance be launched or killed?
+  */
+sealed trait Goal extends Product with Serializable
+
+object Goal {
+
+  /**
+    * There should always be a running Mesos task associated by instance in this state.
```

Comment:
_**Expresses the intent that** there should always be a running Mesos task associated by instance in this state._?

The way it's written atm could be interpreted like it is considered a fact that there's always a task, which is not the case.